### PR TITLE
Justin_Alarm-sidebar-menu-expansion_1203033855288569

### PR DESCRIPTION
**Pull request by pauljustinhernandez**
ASANA ticket: https://app.asana.com/0/1203010733158020/1203033855288569/f

Description:
- added a logic functionality for the sidebar alarm menu which will expand whenever the user clicks the alarm widget

### CODE REVIEW

File: src/app/components/header/header.component.ts [hunk 1]
```diff
@@ -31,16 +32,16 @@ export class HeaderComponent implements OnInit, OnChanges {
...
   constructor(
     private apiService: ApiService,
     private router: Router,
-    private ds: DataService
+    private ds: DataService,
+    private dataCommonService: DataCommonService
   ) {}
 
   ngOnChanges() {
     // this.toggleDisplayDev()
```
Comment (chriscodev) [inline src/app/components/header/header.component.ts:44]: remove commented lines

File: src/app/components/sidebar/sidebar.component.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <aside class="left-sidebar">
     <!-- Sidebar scroll-->
-    <div class="scroll-sidebar ps ps--theme_default ps--active-y" data-ps-id="18c23a0f-542e-d8ac-5865-9b95d267ddbc">
+    <div class="scroll-sidebar ps ps--theme_default ps--active-y" data-ps-id="18c23a0f-542e-d8ac-5865-9b95d267ddbc"
+    [ngStyle]="{'pointer-events' : isSidebarClick ? 'none' : ''}">
```
Comment (chriscodev) [inline src/app/components/sidebar/sidebar.component.html:4]: make this into a function call

**Review by chriscodev — COMMENTED** [review]
Hi I have review comments on this PR

File: src/app/components/header/header.component.ts (hunk 1 above)
Comment (pauljustinhernandez) [reply src/app/components/header/header.component.ts:44]: Will do Tian, thanks!

File: src/app/components/sidebar/sidebar.component.html (hunk 2 above)
Comment (chriscodev) [reply src/app/components/sidebar/sidebar.component.html:4]: yes, this is to make the code shorter and clean in HTML, and all conditional statements should be in .ts file, so when other developer looks into code, they can just refer to the .ts file.

Comment (pauljustinhernandez) [reply src/app/components/sidebar/sidebar.component.html:4]: Ooohh I seee.. Aight Tian, let me refactor it. Will push the updated PR in a bit. Thanks!

**Review by chriscodev — APPROVED** [review]
LGTM

**Pull request merged**